### DISCUSSION TITLE
problem in installing Homebrew in Mac after using this command in official website pg /bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/HEAD/install.sh)"

### DISCUSSION THREAD

**Discussion by VanadiumV** [Homebrew/discussions#3556] [Getting started]
Press RETURN/ENTER to continue or any other key to abort:
==> /usr/bin/sudo /usr/sbin/chown -R l:admin /opt/homebrew
==> Downloading and installing Homebrew...
HEAD is now at d81bd6ab3 Merge pull request #13724 from Homebrew/dependabot/bundler/Library/Homebrew/addressable-2.8.1
error: Not a valid ref: refs/remotes/origin/master
fatal: ambiguous argument 'refs/remotes/origin/master': unknown revision or path not in the working tree.
Use '--' to separate paths from revisions, like this:
'git <command> [<revision>...] -- [<file>...]'

**Comment by SMillerDev** [comment]
See #666
